### ISSUE TITLE
Audio tags are wrong and cant be updated

### ISSUE THREAD

**Issue by H3n-beep** [HardcodeCoder/PulseMusic#113]
Audio tags dont update after they are chabged by a tag editor, and there ia no rescan button and re installing doesnt help
Steps to reproduce
1.install app
2.change audio tags
3.there is no change
**Expected behavior**
Upon closing and opening the app, it is exppected that albums are rebuilt and tags are shown right, oe there ahould be rescan option
**Device and Version Info (please complete the following information):**
 - Device: poco x3 nfc
 - OS: MIUI
 - Android Version:10
 - App Version :latest build on fdroid

**Comment by HardcodeCoder** [comment]
May not be a bug in pulse. Pulse relies completely on Android's system database called `MediaStore`. When a file is editted/deleted/added, the system picks up the changes and automatically rescans the edited file/folder and updates the database. The music player app does not have any role in updating the `MediaStore` in case of externally modified changes. Moreover if any other app is editing a file it is the responsibility of that app to manually update the MediaStore and start a scan for that file (This is applicable for those device where the system does not automatically picks up changes, for example older versions of android sometimes fails to update MediaStore).

With thay said,  MediaStore is known to be buggy at times so a good idea is to do a system restart in case your files are not updated correctly, and this applies to all type of files and not just audio files.

TL;DR Pulse reads whatever data is present in MediaStore and builds upon them. If the database is not up to date pulse cannot show the updated tags.

**Comment by H3n-beep** [comment]
So there ia no way for app to do anything? I tried vlc player and did the changes and vlc understood some of the changes (not all of them) that pulse did not pick up

**Comment by HardcodeCoder** [comment]
I am not sure about VLC (they have a lot of custom implementation and probably extracts metadata from tracks itself rather than MediaStore, in other words their architecture is different). But the fact that even VLC could not pick all of the changes indicates that the tags are not updated correctly.
For now there is no way for pulse to pick up changes unless it is reflected in MediaStore.

**Comment by Matei399YT** [comment]
Why do you close the app is the best
